examples/appengine/hello.go

```diff
@@ -2,10 +2,10 @@ package mellow
 
 import (
 	"fmt"
-	"github.com/bugsnag/bugsnag-go"
-	"github.com/bugsnag/bugsnag-go/errors"
 	"net/http"
 	"os"
+
+	"github.com/bugsnag/bugsnag-go"
```

Comment:
Yeah I have seen community discussion on the package format issue in the past. I think `go fmt` itself will not touch these. My go-fmt separates stdlib from 3rd party packages. Most open source stuff I have seen tends to follow this pattern as well, including libraries like packer, etcd, and kubernetes. ¯\(°_o)/¯